openfisca_web_api_preview/scripts/serve.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import argparse
+from gunicorn.app.base import BaseApplication
+from gunicorn.six import iteritems
+
+from openfisca_core.scripts import add_tax_benefit_system_arguments, detect_country_package
+from ..app import create_app
+
+
+DEFAULT_PORT = '5000'
+BASE_CONF = {
+    'bind': '%s:%s' % ('127.0.0.1', DEFAULT_PORT),
+    'workers': '3',
+}
+
+
+def new_configured_app():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-p', '--port', action = 'store', default = DEFAULT_PORT, help = "port to serve on", type = int)
+    parser.add_argument('-t', '--tracker_url', action = 'store', help = "tracking service url", type = str)
+    parser.add_argument('-s', '--tracker_idsite', action = 'store', help = "tracking service id site", type = int)
```

Comment:
> `-s` for `--tracker_idsite` is not obvious to me.
>
> It would be fine IMO if the two trackers options had no short version (only `--tracker_url` and `--tracker_idsite`)

Reply:
Short version removed.